src/app/joint-control/joint-control-core/motor-settings/motor-settings.component.html

```diff
@@ -141,8 +144,9 @@
     </div>
     <div
         *ngIf="!displayExtended"
-        class="row modal-body h-100 d-flex align-items-start justify-content-center">
-        <div class="col-3 ml-2 mr-2 ">
+        class="row modal-body d-flex align-items-start justify-content-center"
+        style="height: 500px;">
```

Comment:
Why hardcode the pixel values here instead of using bootstrap classes that tend to be more reactive?